substra/__version__.py

```diff
@@ -1 +1 @@
-__version__ = "0.52.0"
+__version__ = "0.53.0"
```

Comment:
Would it not rather be: 
```suggestion
__version__ = "0.52.1"
```